x-pack/plugins/cases/public/components/user_action_tree/user_action_host_isolation_comment_event.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo, useCallback } from 'react';
+import * as i18n from './translations';
+import { LinkAnchor } from '../links';
+import { useKibana } from '../../common/lib/kibana';
+import { SECURITY_SOLUTION_APP_ID } from './constants';
+
+interface EndpointInfo {
+  endpointId: string;
+  hostname: string;
+}
+
+interface Props {
+  type: string;
+  endpoints: EndpointInfo[];
+}
+
+const HostIsolationCommentEventComponent: React.FC<Props> = ({ type, endpoints }) => {
+  const activityLogPath = `/administration/endpoints?selected_endpoint=${endpoints[0].endpointId}&show=activity_log`;
```

Comment:
nit: I think you can get this `/administration/endpoints` path from [here](https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/common/constants.ts#L71) instead of hardcoding it. And if you merge this pr after [this other one ](https://github.com/elastic/kibana/pull/103275)you will be able to get the full path directly from [here](https://github.com/elastic/kibana/pull/103275/files#diff-157daa365722ba58f94ba765e86480456ec7a39409291d27c29cbb1a95e2509dR98)

Reply:
That pr has been merged into master so you can checked in yours now :)